Tests/FluentKitTests/XCTestManifests.swift

```diff
@@ -6,6 +6,11 @@ extension FluentKitTests {
     //   `swift test --generate-linuxmain`
     // to regenerate.
     static let __allTests__FluentKitTests = [
+        ("testCreateEmptyModelArrayDoesntQuery", testCreateEmptyModelArrayDoesntQuery),
```

Comment:
This is outside of the scope of this PR, but `swift test --generate-linuxmain` probably ought to be run by CI

Reply:
Maybe I just like running it myself 🙂